test/sequential/test-repl-persistent-history.js

```diff
@@ -147,6 +150,20 @@ const tests = [{
   test: [UP, UP, UP, CLEAR],
   expected: [prompt, convertMsg, prompt, prompt + '\'=^.^=\'', prompt]
 },
+{
+  before: function() {
+    try {
+      fs.mkdirSync(historyPathFail);
+    } catch (err) {}
```

Comment:
Well, the test will fail if we catch it now that I look at it. I was trying to avoid it hanging around in the event the test throws somewhere.